site/en/docs/privacy-sandbox/ip-protection/index.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'IP Protection'
+subhead: >
+  Improve user privacy by protecting their IP address from being used for tracking.
+description: >
+  A proposal to improve user privacy by protecting their IP address from being used for tracking. 
+date: 2022-12-05
+authors:
+  - anusmitaray
+---
+
+{% Aside %}
+This proposal used to previously have a different scope and was known as Gnatcatcher, or the Global Network Address Translation Combined with Audited and Trusted CDN or HTTP-Proxy Eliminating Reidentification. You can still view the [deprecated documentation](/docs/privacy-sandbox/gnatcatcher/).
+{% endAside %}
+
+## Implementation status
+
+This document outlines a new proposal to help prevent covert tracking: IP Protection.
+
+*  The [IP Protection proposal](https://github.com/spanicker/ip-blindness) has entered [public discussion](https://github.com/spanicker/ip-blindness/issues).
+*  This proposal has not been implemented in any browser.
+*  [The Privacy Sandbox timeline](https://privacysandbox.com/open-web/#the-privacy-sandbox-timeline) provides implementation timings for other Privacy Sandbox proposals. IP Protection is still in its early phase and will have firmer timelines available on this website when available.
+
+## Why do we need this proposal?
+
+IP addresses were created to provide unique identifiers for a client so that traffic could be routed over the internet. IP addresses can be stable over periods of time, which can lead to user identification across first-parties.
```

Comment:
```suggestion
IP addresses were created to provide unique identifiers for a client so that traffic can be routed over the internet. IP addresses can be stable over periods of time, which can lead to user identification across first-parties.
```